models/document-generation.ts

```diff
@@ -557,6 +583,21 @@ const incrementalFundingPlan = {
   },
   additionalProperties: false,
 };
+// TODO FINISH EVAL PLAN OBJECT
+export const evalPlan = {
+  type: "object",
+  properties: {
```

Comment:
Just a heads up if notice on the other document schemas the `additionalProperties: false` is used. Make sure to include it because it helps with the usage of `oneOf` used by the main `generateDocumentSchema` otherwise it won't pass validation.